docs/extensibility/available-slot-fills.md

```diff
@@ -0,0 +1,37 @@
+# Available Slots
+
+To see how to use a slotFill, [check this document](./slot-fills.md).
+
+This is a list of available slots that you can use. If you want to add a new slotFill, [check this document](../../pacakges/checkout/slot/README.md).
+
+## ExperimentalOrderMeta
+This Slot renders below the Checkout summary section and above the "Proceed to Checkout" button in the Cart.
+
+<img width="1135" alt="image" src="https://user-images.githubusercontent.com/6165348/118398683-a0202700-b651-11eb-8a4f-cd8b6ebff53f.png">
+
+### Passed parameters
+
+- `cart`: `wc/store/cart` data but in `camelCase` instead of `snake_case`. [Object breakdown.](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/c00da597efe4c16fcf5481c213d8052ec5df3766/assets/js/type-defs/cart.ts#L172-L188)
+- `extensions`: external data registered by third-party developers using `ExtendRestAPI`. If you used `ExntedRestAPI` on `wc/store/cart` you would find your data under your namespace here.
+
+## ExperimentalOrderShippingPackages
+This slot renders inside the shipping step of Checkout and inside the shipping options in Cart.
+
+Cart:
+
+<img width="1151" alt="image" src="https://user-images.githubusercontent.com/6165348/118399054-2b4dec80-b653-11eb-94a0-989e2e6e362a.png">
+
+Checkout:
+
+<img width="1233" alt="image" src="https://user-images.githubusercontent.com/6165348/118399133-90094700-b653-11eb-8ff0-c917947c199f.png">
+
+### Passed parameters
+
+- `collapsible`: `Boolean` If a shipping package panel should be collapsible or not, this is false in Checkout and true in Cart.
+- `collapse`: `Boolean` If a panel should be collapsed by default, this is true if there's more than 1 fill registred (Core Shipping options are registered as a fill and they're counted).
+- `showItems`: `Boolean` If we should the content of each package, this is true if there's more than 1 fill registred (Core Shipping options are registered as a fill and they're counted).
+- `noResultsMessage`: A react element that you can render if there're no shipping.
```

Comment:
I stumbled a bit as I read through the definition of this parameter.  I made a suggestion below, but I'm not sure if that's exactly what you meant. 🙂

```suggestion
- `noResultsMessage`: A React element that you can render if there are no shipping options.
```

Reply:
It is.